### PR TITLE
feat: better msg when coordinator connection timeout

### PR DESCRIPTION
**Pull request by agatha197**
resolves https://github.com/lablup/giftbox/issues/462

### CODE REVIEW

**Comment by inureyes** [comment]
@agatha197 This PR makes 'adding...' message to 'failed...' without exception.

File: src/components/backend-ai-app-launcher.ts [hunk 1]
```diff
@@ -785,7 +785,10 @@ export default class BackendAiAppLauncher extends BackendAIPage {
       uri = uri + '&is_inference=true';
     }
     uri += '&protocol=' + (servicePortInfo.protocol || 'tcp');
-    this.indicator.set(50, _text('session.launcher.AddingKernelToSocketQueue'));
+    this.indicator.set(
```
Comment (inureyes) [inline src/components/backend-ai-app-launcher.ts]: This change makes every progress message to 'FailedToConnectCoordinator'.

**Review by inureyes — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-app-launcher.ts (hunk 1 above)
Comment (agatha197) [reply src/components/backend-ai-app-launcher.ts]: oops, yes you're right. I modified it to show that's message when only the request failed.

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**